src/Testing/ZopeTestCase/testZopeTestCase.py

```diff
@@ -391,10 +393,9 @@ def testLoggedInUserIsWrapped(self):
 
 
 def test_suite():
-    from unittest import TestSuite
-    from unittest import makeSuite
-    suite = TestSuite()
-    suite.addTest(makeSuite(TestZopeTestCase))
-    suite.addTest(makeSuite(TestPlainUserFolder))
-    suite.addTest(makeSuite(TestWrappingUserFolder))
-    return suite
+    return unittest.TestSuite((
+        unittest.defaultTestLoader.loadTestsFromTestCase(TestZopeTestCase),
+        unittest.defaultTestLoader.loadTestsFromTestCase(TestPlainUserFolder),
+        unittest.defaultTestLoader.loadTestsFromTestCase(
+            TestWrappingUserFolder),
+    ))
```

Comment:
This change looks like you did it manually. Do you know about the feature in `pyupgrade` which automatically does this conversion?
What a pity I did not have the time yet to work on #1031 which could have done this automatically.

Reply:
I did not use `pyupgrade`. I do it when the number of changes is very high or for simple stuff, but in this case I preferred a manual process because I also wanted to clean up how modules are imported etc.

Reply:
P.S.: I personally DO NOT like it when tools like pre-commit force changes. I want the ability to review and change if needed before anything gets committed.